mmv1/products/filestore/api.yaml

```diff
@@ -188,6 +194,7 @@ objects:
                       An integer representing the anonymous group id with a default value of 65534.
                       Anon_gid may only be set with squashMode of ROOT_SQUASH. An error will be returned
                       if this field is specified for other squashMode settings.
+              input: true
```

Comment:
why is this added to `nfsExportOptions` ?

Reply:
I got the latest and rebased, I didn't put this in there.

Reply:
Looks like it was added here: https://github.com/GoogleCloudPlatform/magic-modules/pull/6742/commits/77d15ac22ba7617542cec16be41ec80d5f7d113b#diff-118310b4699d8c11e6cd16d34249e0e1a44f70fbc36f50373bfa1e15d604978dR198 

Seems like you meant it for the `sourceBackup` field, and it probably should have been `output` instead.